### ISSUE TITLE
Users can embed images or videos as part of their question or answer.

### ISSUE THREAD

**Issue by soannie** [Lior-za/Soen341UC3#21]

**Comment by I-Dream-Of-Machines** [comment]
I agree @soannie, it will be an addition to the ask question and answer question functionality.

**Comment by I-Dream-Of-Machines** [comment]
- [ ] Button to add image
- [ ] Button to embed video
- [ ] Function to allow users to upload image/video from their local machine
- [ ] Function that displays image/video as part of the answer
